style-guide.rst

```diff
@@ -0,0 +1,628 @@
+*****************
+Style Guide
+*****************
+
+.. _spelling-and-grammar:
+
+Spelling and grammar
+=======================
+
+.. _american-spelling:
+
+American spelling and grammar
+-----------------------------
+
+Whenever U.S. English and British (or other) English spelling or usage disagree, standard U.S. spelling and usage is preferred.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  The colour of the button is grey
+
+.. rubric:: Right
+
+.. code-block:: rst
+
+  The color of the button is gray.
+  
+.. _quote-marks:
+    
+Quote marks
+--------------
+
+ - Quote marks should generally be avoided if possible.
+ - *Smart* quotes (also known as *curly quotes* or *directional quotes*) are not permitted in source files.
+ 
+.. _avoid-quotes:
+ 
+Avoid quote marks
+~~~~~~~~~~~~~~~~~~~~~
+
+Quote marks are used in prose writing to indicate verbatim text. This is rarely useful in technical writing, as verbatim text usually requires a more specific semantic markup.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  Click the button that says, "Save."
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  Click :guilabel:`Save`.
+  
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  You may see an error message that says, "Something went wrong."
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  You may get an error: ``Something went wrong.``
+
+  
+.. _straight-quote:
+  
+Straight quotes
+~~~~~~~~~~~~~~~~~~
+
+Any time that you *do* need to use quotation marks, use straight (or *plain*) quotes. Sphinx and Docutils will output the typographically correct quote style.
+
+.. _serial-comma:
+
+Serial comma
+-----------------
+
+In a comma-delineated list of items, the penultimate item should be followed by a comma.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  Apples, oranges and pears.
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  Apples, oranges, and pears.
+  
+A bulleted list is often more clear than an inline list.
+
+.. rubric:: Correct
+
+.. code-block:: rst
+
+  You will need to be familiar with git, GitHub, and Python.
+  
+.. rubric:: Possibly Better
+
+.. code-block:: rst
+
+  You will need to be familiar with:
+  
+  - git
+  - GitHub
+  - Python
+  
+There's no hard rule about which to use in any situation. Use your judgement: try it both ways and see which is more clear.
+
+.. _avoid-unneeded-words:
+
+Avoid unneeded words
+-----------------------
+
+.. _adverbs:
+
+Adverbs
+~~~~~~~~~~~
+
+Adverbs often contribute nothing.
+
+Common offenders include:
```

Comment:
I'd remove the previous whitespace and bump this up as a second sentence.

Reply:
ok